app/components/Button/button.css

```diff
@@ -36,6 +36,7 @@
   background: var(--colors-white, #fff);
 }
 
+.secondary.focused,
 .secondary:hover {
   border: 1px solid var(--colors-teal-200, #a6d9d7) !important;
```

Comment:
it shouldn't change the border - it should change the actual icon. So:
* normal - grey icon
* already clicked - teal icon

This should be possible by setting classes on the actual icon. The old icons hide the coloured SVG paths when they have the `focused` class